com.github.dynobo.sphereview.appdata.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <id>com.github.dynobo.sphereview</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <name>SphereView</name>
+  <developer_name>dynobo</developer_name>
+  <summary>Display 360° photospheres and panoramas</summary>
+  <description>
+    <p>Image viewer for 360° equirectangular photospheres and panoramas.</p>
+    <p>Some notable features:</p>
+    <ul>
+      <li>View panoramas and photospheres interactively</li>
+      <li>Lean interface with minimal features</li>
+      <li>100% offline, no telemetry</li>
+      <li>Supports Equirectanglar projection (e.g. used by Google Camera)</li>
+      <li>Supports formats like JPEG, PNG, WEBP</li>
+    </ul>
+    <p>Tags: Photosphere, Panorama, 369, Equirectangular</p>
+  </description>
+  <url type="homepage">https://github.com/dynobo/sphereview</url>
+  <url type="bugtracker">https://github.com/dynobo/sphereview/issues</url>
+  <url type="help">https://github.com/dynobo/sphereview/issues</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/dynobo/sphereview/main/resources/assets/screenshot-1.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/dynobo/sphereview/main/resources/assets/screenshot-2.png</image>
+    </screenshot>
```

Comment:
screenshot links should be from a tag or a commit not `main`.

Reply:
Done in 73f918edb52d7c8340d0e26798cdac4cbb9ef31d